cli/console.py

```diff
@@ -30,7 +33,7 @@ def check_login():
             )
             raise e
 
-        console.print("[italic green] You are logged into Azure. :sparkles:")
+        console.print("[italic green] :sparkles: You are logged into Azure.")
```

Comment:
As Zack has already said, I'm not sure Azure deserves the sparkles, but I think this new interface deserves the sparkles anyway